cfgov/v1/tests/models/test_sublanding_page.py

```diff
@@ -0,0 +1,15 @@
+import mock
+
+from unittest import TestCase
+
+
+class SublandingPageTestCase(TestCase):
+
+    def setUp(self):
+        self.request = mock.MagicMock()
+        self.limit = ''
+        pass
+
+    def test_get_browsefilterable_posts(self):
+
+        pass
```

Comment:
Sorry, don't know how that got in there. That's for a different set of tests I'm working on. I'll get rid of it.